src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -56,11 +56,8 @@ public static class TestOptions
         public static readonly CSharpParseOptions RegularWithRecursivePatterns = Regular8;
         public static readonly CSharpParseOptions RegularWithoutPatternCombinators = Regular8;
         public static readonly CSharpParseOptions RegularWithPatternCombinators = RegularPreview;
-<<<<<<< HEAD
-        public static readonly CSharpParseOptions RegularWithListPatterns = RegularPreview;
-=======
         public static readonly CSharpParseOptions RegularWithExtendedPropertyPatterns = RegularPreview;
->>>>>>> origin/main
+        public static readonly CSharpParseOptions RegularWithListPatterns = RegularPreview;
```

Comment:
nit: this shouldn't be necessary anymore. The default version for all tests has been changed to `preview`.
Only LangVer tests will have to do something explicit (using RegularN for old and TemporaryPreview LangVer for new). TemporaryPreview will be replaced by RegularN+1 when the new language version is introduced.

Reply:
>The default version for all tests has been changed to preview.

Good to know. I can remove in some other PR (existing or upcoming) if that's ok.

Reply:
If you'd like. This is just something to know moving forward. Hopefully it should be a simplification.